go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/go-logr/logr v1.2.2
 	github.com/go-logr/zapr v1.2.3
-	github.com/golang-jwt/jwt v3.2.2+incompatible
```

Comment:
I believe this change incidentally re-introduces a vulnerability fixed in this PR #177. We want to keep the `golang-jwt/jwt` package and remove `dgrijalva/jwt-go`.

Reply:
I corrected this in PR #201